breakdown_internal_test.go

```diff
@@ -86,7 +86,7 @@ func testBreakdownMetricsAlignment(t *testing.T, arch string) {
 		breakdownMetricsMapEntryObj.Type(), false, pkg, "breakdownTiming",
 	)
 	require.NotNil(t, breakdownTimingObj)
-	assert.Equal(t, []int{0}, breakdownTimingFieldIndex)
+	assert.Equal(t, []int{1}, breakdownTimingFieldIndex)
```

Comment:
Had to change the index of the field for the test to pass since I've reordered them.